### PR TITLE
Zeeshan in create polls 36

### PR DESCRIPTION
**Pull request by sk-izsk**
integrate creating polls

### CODE REVIEW

**Review by TheAllen — COMMENTED** [review]
Looks good to me!

File: client/src/App.jsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ const App = () => {
           />
         </Suspense>
       </Switch>
+      <ReactQueryDevtools initialIsOpen={false} />
```
Comment (aeciorc) [inline client/src/App.jsx:34]: What is this for?

File: client/src/pages/SignUp/SignUp.jsx [hunk 2]
```diff
@@ -74,20 +74,15 @@ const SignUp = () => {
...
-          name,
-          email,
-          token,
-        };
-        localStorage.setItem('user', JSON.stringify(userDetails));
+        const { userObject } = data;
+        localStorage.setItem('user', JSON.stringify(userObject));
         reset();
+        history.push('/home');
+        window.location.reload();
```
Comment (aeciorc) [inline client/src/pages/SignUp/SignUp.jsx]: don't need this

File: client/src/components/Header/Header.jsx [hunk 3]
```diff
@@ -36,17 +39,29 @@ const useStyles = makeStyles((theme) => ({
...
+  console.log('cookie', cookies);
   const [openFriends, setOpenFriends] = useState(false);
 
   function handleFriendsModal() {
     setOpenFriends(!openFriends);
   }
   const handlePollModal = () => setOpenPoll(!openPoll);
+  const handleLogOut = () => {
+    localStorage.removeItem('user');
+    window.location.reload();
```
Comment (aeciorc) [inline client/src/components/Header/Header.jsx:63]: neither this

File: client/src/App.jsx (hunk 1 above)
Comment (sk-izsk) [reply client/src/App.jsx:34]: this is dev tools from a library called react query to checko whether i am getting data properly or not, avoid console long. dont worry it wont pop up in production

File: client/src/pages/SignUp/SignUp.jsx (hunk 2 above)
Comment (sk-izsk) [reply client/src/pages/SignUp/SignUp.jsx]: oh yes I forget thanks

File: client/src/components/Header/Header.jsx (hunk 3 above)
Comment (sk-izsk) [reply client/src/components/Header/Header.jsx:63]: yeap thanks

File: client/src/App.jsx (hunk 1 above)
Comment (aeciorc) [reply client/src/App.jsx:34]: Ok

Comment (sk-izsk) [reply client/src/App.jsx:34]: easy to track what data i am fetching thats why use it

Screenshot 2020-10-29 160659

File: client/src/App.jsx [hunk 4]
```diff
@@ -17,9 +18,9 @@ const App = () => {
   return (
     <>
       {!!stateContext.user && <Header />}
-      <Header />
       <Switch>
         <Suspense fallback={<LoadingScreen />}>
+          <Redirect to={!!stateContext.user ? '/home' : '/login'} />
```
Comment (aeciorc) [inline client/src/App.jsx:23]: This doesn't look right. What if the user go to a different route within the dashboard? @SsaaaadD2 this is what you mentioned, right?

Comment (sk-izsk) [reply client/src/App.jsx:23]: Hmm so for now lets delete it and create different ticket for routing then ?? @aeciorc

Comment (aeciorc) [reply client/src/App.jsx:23]: I think @SsaaaadD2  did adress that in his other branch

Comment (sk-izsk) [reply client/src/App.jsx:23]: ok so for now then should I delete this or keep it ?

Comment (aeciorc) [reply client/src/App.jsx:23]: Just keep it

**Review by aeciorc — COMMENTED** [review]
Merge dev into it and solve the conflicts, then I'll approve it

**Comment by sk-izsk** [comment]
> Merge dev into it and solve the conflicts, then I'll approve it

i was ready with `git push` command to wait for your reply :P

File: server/routes/polls/polls.js [hunk 5]
```diff
@@ -30,6 +30,9 @@ router.post('/polls', authentication, async (req, res) => {
       const response = await PollsMongoModel.find({ userId });
       res.status(200).json(response);
     });
+    // .then((response) => {
+    //   res.status(200).json(response);
+    // });
```
Comment (aeciorc) [inline server/routes/polls/polls.js]: Remove this, right?

Comment (sk-izsk) [reply server/routes/polls/polls.js]: yeap my bad its removed

File: client/src/App.jsx (hunk 4 above)
Comment (SsaaaadD2) [reply client/src/App.jsx:23]: Yes I changed this in the other branch

**Review by aeciorc — APPROVED** [review]

**Pull request merged**